src/components/charts/PercentageStackedBarChart.jsx

```diff
@@ -0,0 +1,166 @@
+import {
+  BarChart, Bar, Tooltip, XAxis, YAxis,
+  CartesianGrid, LabelList,
+} from 'recharts';
+import PropTypes from 'prop-types';
+import React from 'react';
+import _ from 'underscore';
+import styled from 'styled-components';
+
+const getPercentageData = (chartData, percentageFixedPoint) => {
```

Comment:
also very similar to the functions in SummaryPieChart and SummaryBarChart - I would move into a global helper function